project/Dependencies.scala

```diff
@@ -15,21 +15,22 @@ object Dependencies {
   lazy val java8CompatVersion = settingKey[String]("The version of scala-java8-compat to use.")
 
   val junitVersion = "4.12"
-  val slf4jVersion = "1.7.25"
+  val slf4jVersion = "1.7.28"
   // check agrona version when updating this
-  val aeronVersion = "1.19.1"
+  val aeronVersion = "1.21.2"
   // needs to be inline with the aeron version
-  val agronaVersion = "1.0.1"
+  val agronaVersion = "1.0.7"
```

Comment:
(important to doublecheck)

Reply:
that is the right one, together with Aeron 1.21.2 👍